src/base/jstemplates/pages-nav.html

```diff
@@ -10,3 +10,11 @@
     {{/ pages }}
   </ul>
 </nav>
+
+<nav class="list-toggle-nav">
+  <a class="list-toggle-btn btn btn-block">
+  	<span class="show-the-map is-visuallyhidden">{{ show_map_button_label }}</span>
+  	<span class="is-screen-reader-text">/</span>
```

Comment:
This block of code was relocated from `base.html` as-is. It already had that slash, and I'm not totally sure why. Maybe just so the screen reader span has some content in it?